src/style.css

```diff
@@ -1,3 +1,3 @@
 body {
-    background-color: bisque;
+  background-color: bisque;
 }
```

Comment:
- [ ] I would suggest keeping the design similar to the given wireframe. Currently, Your design is not matching with the given one.

### Your design

![Screenshot from 2021-12-14 15-11-51](https://user-images.githubusercontent.com/22434660/145973740-b88319e1-ebe9-4131-a85e-b846b5b52093.png)


### Given design

![Screenshot from 2021-12-14 15-11-41](https://user-images.githubusercontent.com/22434660/145973714-646f89e2-4f13-4971-9e70-54091bafc827.png)

Reply:
Hello, the requirements specifically says that no design should be made for that first part of the exercise....